common/predictive-text/unit_tests/headless/lmlayer.js

```diff
@@ -1,7 +1,7 @@
 var assert = require('chai').assert;
 var sinon = require('sinon');
 
-let LMLayer = require('../../');
```

Comment:
This can be left as is if `"main"` in `package.json` is changed to `build/index.js`.

Reply:
Huh.  Wish I understood how that part works; I'd make the change if I could do something equivalent for the worker-initialization headless test case, but it doesn't seem to generalize to sub-paths.